model/atmosphere/diffusion/tests/diffusion/integration_tests/test_benchmark_diffusion.py

```diff
@@ -0,0 +1,228 @@
+# ICON4Py - ICON inspired code in Python and GT4Py
+#
+# Copyright (c) 2022-2024, ETH Zurich and MeteoSwiss
+# All rights reserved.
+#
+# Please, refer to the LICENSE file in the root directory.
+# SPDX-License-Identifier: BSD-3-Clause
+
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any
+
+import gt4py.next as gtx
+import pytest
+
+
+if TYPE_CHECKING:
+    import gt4py.next.typing as gtx_typing
+import icon4py.model.common.dimension as dims
+import icon4py.model.common.grid.states as grid_states
+from icon4py.model.atmosphere.diffusion import diffusion, diffusion_states
+from icon4py.model.common.constants import RayleighType
+from icon4py.model.common.grid import (
+    geometry as grid_geometry,
+    geometry_attributes as geometry_meta,
+    vertical as v_grid,
+)
+from icon4py.model.common.initialization.jablonowski_williamson_topography import (
+    jablonowski_williamson_topography,
+)
+from icon4py.model.common.interpolation import interpolation_attributes, interpolation_factory
+from icon4py.model.common.metrics import metrics_attributes, metrics_factory
+from icon4py.model.common.states import prognostic_state as prognostics
+from icon4py.model.common.utils import data_allocation as data_alloc
+from icon4py.model.testing import definitions, grid_utils
+from icon4py.model.testing.grid_utils import construct_decomposition_info
+
+from ..fixtures import *  # noqa: F403
+
+
+@pytest.mark.embedded_remap_error
+@pytest.mark.benchmark(
+    group="diffusion_benchmark",
+)
+@pytest.mark.parametrize(
+    "grid", [definitions.Grids.MCH_OPR_R19B08_DOMAIN01, definitions.Grids.R02B07_GLOBAL]
+)
```

Comment:
Maybe we can introduce a fixture `benchmark_grid` that we can then re-use in other benchmark cases.

Reply:
you can also do it latest when we need it the second time, so for the dycore... @havogt on what grids are the StencilTest benchmark run? Still the validation grids?

Reply:
Yes, ok to do it in the next PR. For the stencil tests we use the `MCH_OPR_R19B08_DOMAIN01` via `--grid benchmark_grid`.